### DISCUSSION TITLE
Reading group access to private/draft mode works

### DISCUSSION THREAD

**Discussion by crowesn** [ManifoldScholar/manifold#3495] [Q&A]
`As an` author, 
`I want to` grant access to a reading group to my private/draft work.
`So that I can` gather feedback/review via annotations before publishing the work.

We'd like to see a feature added where reading groups can be granted access to draft works. Otherwise restricted works would be accessible based on reading group membership.

**Comment by zdavis** [comment]
You can grant a project entitlement to a reading group and set the project to closed access, which I think will accomplish what you want here.
